pom.xml

```diff
@@ -189,6 +189,47 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.revapi</groupId>
+          <artifactId>revapi-maven-plugin</artifactId>
```

Comment:
After consulting with psiroky, we decided to move it to kie-parent. I will change it tomorrow.